kythe/cxx/verifier/assertion_ast.h

```diff
@@ -42,6 +43,20 @@ class SymbolTable {
  public:
   explicit SymbolTable() : id_regex_("[%#]?[_a-zA-Z/][a-zA-Z_0-9/]*") {}
 
+  /// \brief Returns the `Symbol` associated with `string` or `nullopt`.
+  absl::optional<Symbol> MaybeIntern(absl::string_view string) const {
```

Comment:
`FindInterned`? using `Intern` as the very suggests that the string value will be interned, not just looked up